src/OrchardCore.Modules/OrchardCore.Forms/OrchardCore.Forms.csproj

```diff
@@ -11,10 +11,26 @@
     <PackageTags>$(PackageTags) OrchardCoreCMS</PackageTags>
   </PropertyGroup>
 
+  <ItemGroup>
+    <EmbeddedResource Remove="Controllers\ValidationRuleController.cs~RF3440db28.TMP" />
+    <EmbeddedResource Remove="Providers\IsBeforeProvider.cs~RF34fe94ad.TMP" />
+    <EmbeddedResource Remove="wwwroot\Scripts\validationRuleHelper.js" />
+    <EmbeddedResource Remove="wwwroot\Scripts\validationRuleHelper.min.js" />
+  </ItemGroup>
+
   <ItemGroup>
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Include="wwwroot\Scripts\validationRuleHelper.js">
+      <CopyToPublishDirectory>Never</CopyToPublishDirectory>
+    </None>
+    <None Include="wwwroot\Scripts\validationRuleHelper.min.js">
+      <CopyToPublishDirectory>Never</CopyToPublishDirectory>
+    </None>
+  </ItemGroup>
+
```

Comment:
Yes, remove all these extra item groups, we have msbuild scripts that embeds resources including those under wwwroot

Reply:
ok